sdk/eventgrid/azure-eventgrid/swagger/postprocess_eventnames.py

```diff
@@ -0,0 +1,54 @@
+import inspect
+import re
+import warnings
+import sys
+from azure.eventgrid._generated import models
+
+backward_compat = {
+    'AcsChatMemberAddedToThreadWithUserEventName': "Microsoft.Communication.ChatMemberAddedToThreadWithUser",
+    'ResourceWriteFailureEventName': "Microsoft.Resources.ResourceWriteFailure",
+    'IoTHubDeviceDeletedEventName': "Microsoft.Devices.DeviceDeleted",
+    'IoTHubDeviceDisconnectedEventName': "Microsoft.Devices.DeviceDisconnected",
+    'ResourceDeleteFailureEventName': "Microsoft.Resources.ResourceDeleteFailure",
+    'ResourceDeleteCancelEventName': "Microsoft.Resources.ResourceDeleteCancel",
+    'AcsChatThreadParticipantAddedEventName': "Microsoft.Communication.ChatThreadParticipantAdded",
+    'ResourceDeleteSuccessEventName': "Microsoft.Resources.ResourceDeleteSuccess",
+    'EventGridSubscriptionValidationEventName': "Microsoft.EventGrid.SubscriptionValidationEvent",
+    'ResourceWriteSuccessEventName': "Microsoft.Resources.ResourceWriteSuccess",
+    'ResourceActionSuccessEventName': "Microsoft.Resources.ResourceActionSuccess",
+    'ResourceWriteCancelEventName': "Microsoft.Resources.ResourceWriteCancel",
+    'ServiceBusDeadletterMessagesAvailableWithNoListenerEventName': "Microsoft.ServiceBus.DeadletterMessagesAvailableWithNoListener",
+    'ResourceActionFailureEventName': "Microsoft.Resources.ResourceActionFailure",
+    'AcsChatMemberRemovedFromThreadWithUserEventName': "Microsoft.Communication.ChatMemberRemovedFromThreadWithUser",
+    'IoTHubDeviceConnectedEventName': "Microsoft.Devices.DeviceConnected",
+    'EventGridSubscriptionDeletedEventName': "Microsoft.EventGrid.SubscriptionDeletedEvent",
+    'AcsChatThreadParticipantRemovedEventName': "Microsoft.Communication.ChatThreadParticipantRemoved",
+    'ResourceActionCancelEventName': "Microsoft.Resources.ResourceActionCancel",
+    'IoTHubDeviceCreatedEventName': "Microsoft.Devices.DeviceCreated"
+}
+
+def event_tuples(system_events):
+    tup_list = []
+    for event in system_events:
+        class_name = event[0].replace("Data", "Name")
+        try:
+            event_name = re.findall("Microsoft.[a-zA-Z]+.[a-zA-Z]+", event[1].__doc__)[0]
+        except:
```

Comment:
Do we want to catch all types exceptions here?

Reply:
yes - since this script is to autogenerate the evenname mappings, we dont want to let anything slip by